### ISSUE TITLE
Using alongside OneTab (and no other extension as all were eliminated/disabled) Save as no  longer "works" in the browser of choice.

### ISSUE THREAD

**Issue by Turk303** [gyng/save-in#153]
As title. Pre warning and apology, due to so many variables I doubt any info I supply will help. *

1 thought was perhaps the huge amount of One Tab saves had started an issue/clash with Save In but I have synced One Tab in Vavaldi and Save As works fine in THAT browser.

Current browser (CheDot) with all other extensions disabled the 2 ext's don't work.I do have a lot of saved tabs in One Tab but.. ?

They (extensions) both worked fine together and then (likely magic or pixies or there's a slight chance of me forgetting what i changed and they stopped working together but for the life of me (i blame pixies) I can not recall what (started about a month ago, though I'd fix the issue, some how, never happened. - 

It possibly can't be OS related as I've just had to reinstal Windows 10 thanks to an update that hung and (yep pixies again!) so theres more useless info thats not helping you pin point the problem.

. . . . 

Despite only 2 extensions "on" in chedot something is causing a clash and it's not the amount of tabs saved, & theoretically it's not chedot as the 2 extensions worked fine before. (as you know, both vivaldi & chedot are chrome cores browsers....

I iwsh I could say "I did this  n that and now no worky" but gibberish, pixies and no worky not to helpful eh?

I'll attmpt some cleaning up of chedot (remove unused/on hold extensions + now I know vivaldi has the one tabs synced, Ill uninstal/reinstal that extension.

Does "Save In" leave anything upon removal that i can physically delete?
IN the meantime If you could suggest sumfink?

K, if yer still awake / or not scooped your eyes out with a teaspoon out of boredom cheers for the  tool & cheers for any help.

ps apologise for calling it Save As, here n there....

**Comment by gyng** [comment]
Hello, if you are using Chedot browser and not Firefox/Chrome, I can't provide any support. Do the extensions work together in Firefox or Chrome?

**Comment by Turk303** [comment]
Seems odd not being able to help when chrome based browsers can use ALL the
google supplied extensions.
I'm not being rude but thats a bit of a cop out.
But to answer your question, Yes they work in Chrome (& Brave & Vivaldi &
chedot, until I tried to edit the scripts in chedot, then it broke, so i
removed, cleared caches, reset and reinstalled and it's still broken in
chedot but working fine under the same circumstances regardless of chrome
based browsers name.

Whether its running in chrome or brave or vivaldi or chedot being a chrome
based browser, just like the other 10s of thousands of extensions they all
work in chrome based browsers, I'm certain you know this.

NVM I've found Pantherbar - it doesn't need scripting as it comes with pre
made ones.

Thanks, for the reply.
